lab-megan/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const greet = require('./lib/greetings.js');
+
+process.argv.forEach((val, index) => {
+  console.log(`${index}: ${val}`);
+});
+
+greet.sayHello('Scott');
+greet.sayGoodbye();
```

Comment:
Glad to see you used process.argv[2] for the bonus!

Another way you could have implemented this was to attach another method onto the exports object in greetings.js. This method could have a variable set to process.argv[2] that could be passed as an argument which would dynamically generate a name for you. You could then invoke that method on index.js, similarly to how you are already invoking the others :)